src/runner/program/python.rs

```diff
@@ -0,0 +1,27 @@
+use super::*;
+pub struct PythonProgram(String);
+    
+impl Program for PythonProgram {
+    fn run(&self, code_block: &CodeBlock) -> Result<std::process::Output, std::io::Error> {
+        use std::io::Write;
+        let mut process = std::process::Command::new(&self.0);
+        process
+            .arg("-")
+            .stdin(std::process::Stdio::piped());
+        let mut child = process.spawn()?;
+        child.stdin.take().expect("Failed to get stdin of python").write_all(code_block.code.as_bytes())?;
+        Ok(child.wait_with_output()?)
+    }
+}
+impl PythonProgram {
+    pub(super) fn search() -> SearchStatus {
+        for shell in ["python3", "python"] {
```

Comment:
⚠️ `python` can sometimes refer to python 2 ⚠️

Reply:
Yes I know but what should i do ? Not using py2 at all, only py3 ? Check the `python` version ?

Reply:
I think python 3 is backwards compatible, try prioritizing python 3 interpreter if possible? 🤔
Check python version and put a warning ?

Reply:
I implement a chack for version >=3 in [5239d73](https://github.com/glcraft/aio/pull/22/commits/5239d73d804a11db96b0a36f496ad8a06ffe6420)